rpmlb/cli.py

```diff
@@ -73,6 +73,22 @@
     '--copr-repo', '-C', metavar='COPR_REPO',
     help='Target Copr for copr builder.',
 )
+@click.option(
+    '--epel', metavar='VERSION', type=int, default=7,
+    help='Target EPEL version for koji builder.',
+)
+@click.option(
+    '--koji-profile', metavar='PROFILE',
+    help='Configuration profile for koji builder.',
+)
+@click.option(
+    '--koji-target-format', metavar='FORMAT',
+    help='Target format string for koji builder.',
+)
+@click.option(
+    '--scratch', is_flag=True, default=False,
```

Comment:
Good point – not all builders would support scratch builds. To be consistent with others, I will use `koji-scratch`.